societe-francaise-degyptologie.csl

```diff
@@ -0,0 +1,212 @@
+          </if>
+          <else-if match="any" position="ibid-with-locator">
+            <choose>
+              <if match="any" disambiguate="true">
+                <text macro="disambiguate-citation"/>
+              </if>
+              <else>
+                <text term="ibid" form="short" font-style="italic"/>
+              </else>
+            </choose>
+          </else-if>
+          <else-if match="any" position="subsequent">
+            <choose>
+              <if match="any" disambiguate="true">
+                <text macro="disambiguate-citation"/>
+              </if>
+              <else>
+                <text value="op. cit." font-style="italic" text-decoration="none"/>
+              </else>
+            </choose>
+          </else-if>
+          <else>
+            <choose>
+              <if type="chapter" match="any">
+                <group delimiter=" ">
+                  <text term="in"/>
+                  <text macro="editor"/>
+                </group>
+              </if>
+            </choose>
+            <choose>
+              <if type="article article-journal article-magazine article-newspaper" match="any">
+                <text macro="title"/>
+                <group delimiter=" ">
+                  <text macro="container-title"/>
+                  <text macro="volume"/>
+                  <text macro="date" prefix="(" suffix=")"/>
+                </group>
+              </if>
+              <else-if match="any" variable="volume">
+                <choose>
+                  <if type="book" match="any">
+                    <text macro="title"/>
+                  </if>
+                  <else-if type="chapter" match="any">
+                    <text macro="container-title"/>
+                  </else-if>
+                </choose>
+                <group delimiter=" ">
+                  <text macro="volume"/>
+                  <choose>
+                    <if match="any" variable="collection-title">
+                      <group delimiter=" " prefix="(" suffix=")">
+                        <text macro="collection"/>
+                        <text macro="collection-number"/>
+                      </group>
+                    </if>
+                  </choose>
+                </group>
+                <text macro="date"/>
+              </else-if>
+              <else>
+                <group delimiter=" ">
+                  <choose>
+                    <if type="book" match="any">
+                      <text macro="title"/>
+                    </if>
+                    <else-if type="chapter" match="any">
+                      <text macro="container-title"/>
+                    </else-if>
+                  </choose>
+                  <group delimiter=" " prefix="(" suffix=")">
+                    <text macro="collection"/>
+                    <text macro="collection-number"/>
+                  </group>
+                </group>
+                <text macro="date"/>
```

Comment:
There is no space or other delimiter coming between the container information and the date. I'm guessing that's not right. 

You add a group with the appropriate delimiter around this container+date section and the other places above

Reply:
There was missing a group around lines 162-164, a space should appear now